package.json

```diff
@@ -45,9 +45,9 @@
   "scripts": {
     "all": "npm run clean && npm run build && npm run lint && npm run test",
     "postinstall": "lerna bootstrap --no-ci && link-parent-bin",
-    "lint": "eslint . --ext .ts,.tsx",
-    "lint:log": "eslint . --ext .ts,.tsx -f compact -o lint.log",
-    "lint:fix": "eslint . --ext .ts,.tsx --fix",
+    "lint": "eslint . --resolve-plugins-relative-to . --ext .ts,.tsx",
```

Comment:
Why is `--resolve-plugins-relative-to` needed. What does it do?

Reply:
`fix issues with linting (duplication of configurations, see: eslint/eslint#13385)`
{ extends: "../../tslint.config.json" (or whatever) } doesnt work in newer versions of eslint (7.2.0, tho for now we should stay with 6.8.0 because import plugin doesnt work with newest)